components/mpas-ocean/src/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -214,6 +221,23 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, layerThickness, layerThickEdgeMea
             use_Redi_diag_terms = .false.
          endif
 
+         if (use_quasi_monotone) then
+            minimumBnd(:, :, iCell) = 0.0_RKIND
+            maximumBnd(:, :, iCell) = 0.0_RKIND
+            do k = minLevelCell(iCell), maxLevelCell(iCell)
```

Comment:
will need to add new variables to the OMP private directives abpve

Reply:
The `bnd` arrays should be shared I believe, but I added the indexing `kUp`, `kDn`, etc --- thanks!